src/examples/ex02_root.ts

```diff
@@ -15,9 +24,24 @@ class Main extends Circuit {
   }
 }
 
+console.log('generating keypair...');
```

Comment:
To be honest - I run this example file often for testing, and I like them because the whole operation takes more than 10 seconds and this waiting time is less confusing/annoying if you see what is currently happening. We could remove the timings